src/Time/Time.test.js

```diff
@@ -610,30 +611,24 @@ describe('<Time />', () => {
         expect(wrapper.state('time').minute).toEqual('00');
         expect(wrapper.state('time').second).toEqual('00');
 
-        wrapper = mount(
-            <Time
-                format12Hours
-                name='meridiem'
-                time={{ hour: 1, minute: 0, second: 0, meridiem: 0 }} />
-        ).children().children();
-        expect(wrapper.state('time').second).toEqual(0);
+        // expect(wrapper.state('time').second).toEqual(0);
         wrapper
             .find(
                 'button.fd-button--light.sap-icon--navigation-up-arrow'
             )
             .at(2)
             .simulate('click');
-        expect(wrapper.state('time').hour).toEqual(1);
-        expect(wrapper.state('time').minute).toEqual(0);
+        expect(wrapper.state('time').hour).toEqual('01');
+        expect(wrapper.state('time').minute).toEqual('00');
         expect(wrapper.state('time').second).toEqual('01');
```

Comment:
I don't want to keep this change in this pr, but just included it to get some context. Why are the tests expecting strings sometimes, numbers other times?? See commented out test on line 614 and test on 613 - testing the same thing except string versus number. How is this even happening?